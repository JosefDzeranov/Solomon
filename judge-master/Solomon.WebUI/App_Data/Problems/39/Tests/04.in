500 100
1 2 4 5 8 10 11 12 13 14 17 18 19 26 30 36 37 41 46 47 48 50 52 54 56 57 59 60 61 63 65 66 67 68 72 75 77 78 83 84 85 86 87 88 91 92 94 96 97 99 103 106 109 113 114 115 116 118 119 124 126 128 132 134 137 138 141 142 143 146 147 152 156 157 158 159 160 163 164 165 166 167 169 174 175 177 179 180 181 182 183 184 185 188 189 192 194 195 197 200 