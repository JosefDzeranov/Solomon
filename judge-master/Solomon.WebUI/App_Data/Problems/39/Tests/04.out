1-2 4-5 8 10-14 17-19 26 30 36-37 41 46-48 50 52 54 56-57 59-61 63 65-68 72 75 77-78 83-88 91-92 94 96-97 99 103 106 109 113-116 118-119 124 126 128 132 134 137-138 141-143 146-147 152 156-160 163-167 169 174-175 177 179-185 188-189 192 194-195 197 200 